lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -966,8 +966,8 @@ instance Arbitrary ApiWalletDelegationNext where
             <$> fmap Just arbitrary
             <*> fmap Just arbitrary
         , ApiWalletDelegationNext Api.NotDelegating
-            <$> pure Nothing
-            <*> fmap Just arbitrary
+            Nothing
+            . Just <$> arbitrary
```

Comment:
These two lines could probably be coalesced into a single line.

Reply:
Fixed, thanks